### PR TITLE
fix transfer accounts bug

### PR DESCRIPTION
**Pull request by peterQin0826**
## What does this implement/fix? Explain your changes.
fix transfer accounts bug and fix watch wallet ui bug
## Does this close any currently open issues?  
If it fixes a bug or resolves a feature request, be sure to link to that issue.
OneKeyHQ/TaskHub#518,OneKeyHQ/TaskHub#517,OneKeyHQ/TaskHub#382
## Pull request type
 
_Put an `x` in the boxes that apply_
- [ x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## Where has this been tested?
…

## Any other comments?
…

### CODE REVIEW

File: android/app/src/main/java/org/haobtc/onekey/activities/sign/SignActivity.java [hunk 1]
```diff
@@ -433,6 +434,7 @@ public void onGotPass(GotPassEvent event) {
     /**
      * 扫描二维码
      */
+    @SuppressLint("CheckResult")
```
Comment (danielsocials) [inline android/app/src/main/java/org/haobtc/onekey/activities/sign/SignActivity.java:437]: 忽略lint的原因是？

File: android/app/src/main/java/org/haobtc/onekey/onekeys/homepage/process/SendHdActivity.java [hunk 2]
```diff
@@ -822,17 +823,15 @@ private void keyBoardHideRefresh() {
...
-                if (decimal.compareTo(minAmount) <= 0) {
-                    String min = String.format(Locale.ENGLISH, "%s", minAmount.stripTrailingZeros().toPlainString());
-                    editAmount.setText(min);
-                    editAmount.setSelection(min.length());
-                } else {
-                    if (addressInvalid) {
-                        calculateMaxSpendableAmount();
-                    }
+            if (!Strings.isNullOrEmpty(amount) && Double.parseDouble(amount) > 0) {
+//                BigDecimal decimal = BigDecimal.valueOf(Double.parseDouble(amount));
```
Comment (danielsocials) [inline android/app/src/main/java/org/haobtc/onekey/onekeys/homepage/process/SendHdActivity.java:827]: delete line

File: android/app/src/main/java/org/haobtc/onekey/utils/CoinUtils.java [hunk 3]
```diff
@@ -15,12 +12,12 @@ private CoinUtils() {
 
     public static List<CoinBean> getSupportCoins() {
         CoinBean btc = new CoinBean(R.drawable.token_btc, R.string.coin_btc);
-        CoinBean eth = new CoinBean(R.drawable.token_eth, R.string.coin_eth);
-        CoinBean eos = new CoinBean(R.drawable.token_eos, R.string.coin_eos);
+//        CoinBean eth = new CoinBean(R.drawable.token_eth, R.string.coin_eth);
+//        CoinBean eos = new CoinBean(R.drawable.token_eos, R.string.coin_eos);
```
Comment (danielsocials) [inline android/app/src/main/java/org/haobtc/onekey/utils/CoinUtils.java:16]: delete

File: android/app/src/main/java/org/haobtc/onekey/utils/CoinUtils.java [hunk 4]
```diff
@@ -15,12 +12,12 @@ private CoinUtils() {
...
         List<CoinBean> list = new ArrayList<>();
         list.add(btc);
-        list.add(eth);
-        list.add(eos);
+//        list.add(eth);
```
Comment (danielsocials) [inline android/app/src/main/java/org/haobtc/onekey/utils/CoinUtils.java:19]: delete

File: android/app/src/main/java/org/haobtc/onekey/activities/sign/SignActivity.java (hunk 1 above)
Comment (peterQin0826) [reply android/app/src/main/java/org/haobtc/onekey/activities/sign/SignActivity.java:437]: idea 提示的有警告，推荐添加的，已修改

File: android/app/src/main/java/org/haobtc/onekey/onekeys/homepage/process/SendHdActivity.java (hunk 2 above)
Comment (peterQin0826) [reply android/app/src/main/java/org/haobtc/onekey/onekeys/homepage/process/SendHdActivity.java:827]: 已修改

File: android/app/src/main/java/org/haobtc/onekey/utils/CoinUtils.java (hunk 3 above)
Comment (peterQin0826) [reply android/app/src/main/java/org/haobtc/onekey/utils/CoinUtils.java:16]: 已修改

File: android/app/src/main/java/org/haobtc/onekey/utils/CoinUtils.java (hunk 4 above)
Comment (peterQin0826) [reply android/app/src/main/java/org/haobtc/onekey/utils/CoinUtils.java:19]: 已修改

File: android/app/src/main/java/org/haobtc/onekey/activities/sign/CheckSignActivity.java [hunk 5]
```diff
@@ -168,6 +169,7 @@ public void onViewClicked(View view) {
         }
     }
 
+    @SuppressLint("CheckResult")
```
Comment (danielsocials) [inline android/app/src/main/java/org/haobtc/onekey/activities/sign/CheckSignActivity.java:172]: 没有合理的理由，不应该屏蔽lint结果

**Pull request merged**